src/ts/app/view/components/ConfirmModal.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react'
+import { Modal } from './Modal'
+
+function acceptOrReject(acceptReject: (() => void) | undefined, close: () => void): void {
```

Comment:
maybe more natural to make `acceptReject` optional (which means we have to put it last)
```ts
function acceptOrReject(close: () => void, acceptReject?: () => void)): void 
```